internal/livechat/api.go

```diff
@@ -14,27 +14,41 @@ import (
 
 const BillingAPIBaseURL = "https://billing.livechatinc.com"
 
+const (
+	ChargeStatusPending    ChargeStatus = "pending"
+	ChargeStatusSuccess    ChargeStatus = "success"
+	ChargeStatusActive     ChargeStatus = "active"
+	ChargeStatusProcessing ChargeStatus = "processing"
+	ChargeStatusCancelled  ChargeStatus = "cancelled"
+	ChargeStatusFailed     ChargeStatus = "failed"
+	ChargeStatusDeclined   ChargeStatus = "declined"
+	ChargeStatusFrozen     ChargeStatus = "frozen"
```

Comment:
Some statuses are only for direct or recurring payments. Maybe it should be reflected in the names?